newpotato/hitl.py

```diff
@@ -192,20 +293,27 @@ def store_triplet(self, text: str, pred: Tuple[int, ...], args: List[Tuple[int,
             return self.store_triplet(self.latest, pred, args)
         assert self.is_parsed(text), f"unparsed text: {text}"
         logging.info(f"appending to triplets: {pred}, {args}")
-        self.triplets[text].append((pred, args))
+        self.text_to_triplets[text].append(Triplet(pred, args))
 
-    def classify(self, graph: Hyperedge) -> List[Dict[str, Any]]:
+    def extract_triplets_from_text(self, text: str) -> Dict[str, Any]:
         """
-        Classify the graph.
+        Extract the triplets from the given text with the Extractor.
+        First the text is parsed into graphs, then the graphs are classified by the Extractor.
 
         Args:
-            graph (Hyperedge): The graph to classify.
+            text (str): The text to extract triplets from.
 
         Returns:
-            List[Dict[str, Any]]: The matches in a format of [{"REL": "pred", "ARG1": "arg1", "ARG2": "arg2"}]
+            Dict[str, Any]: The matches in a format of {"text": [{"REL": "pred", "ARG1": "arg1", "ARG2": "arg2"}]}
```

Comment:
Is this really still what it returns? There now  seem to be "matches" and "rules_triggered" fields. Also I think nothing requires that there are only two arguments.